client/landing/stepper/declarative-flow/new-hosted-site-flow.ts

```diff
@@ -89,6 +96,14 @@ const hosting: Flow = {
 						extra: { geo_affinity: siteGeoAffinity },
 					} );
 
+					if ( productSlug === 'wp_bundle_hosting_trial_monthly' ) {
```

Comment:
Let's use the constant: 

```suggestion
					if ( productSlug === PLAN_HOSTING_TRIAL_MONTHLY ) {
```